Clinical-Genomics/scout
# Associate institutes with multiple loqusdb instances

- Associate an institute to more than one loqusdb instance. Choose instances from institute settings
- On variant pages, display occurrences from several loqusdb instances
- No need to name one of the locusdb instances "default" in scout config file
-  FIx #1617 and fix #1830. 


<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Deploy master branch on cg-vm1 and go to [a variant with observations in other cases](https://scout-stage.scilifelab.se/cust000/SWEDAC-2019-HG-05-Trio-NovaSeq/7076347d9ff7b06e8c24104ef14a2152)
1. Deploy this branch on cg-vm1 and make sure the variant above shows the same info (cases should be more since I've fixed an error in the code, see my comment in the code)
2. Modify the scout config file of cg-vm1 to have a 2 functioning instances of loqusdbapi (could be the same repeated twice as well)
3. Go to an institute's settings and select both instances on the dropdown menu. Save.
4. Go to the variant above and make sure stats for both loqusdb instances are visible

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
